submodules/vmbox/vmbox.js

```diff
@@ -282,9 +282,9 @@ define(function(require) {
 					return re.test(email);
 				},
 				getRecipients = function() {
-					var list = $('#recipients_list', vmbox_html).val().replace(/\s+/g, '').split(',');
+					var list = $('#recipients_list', vmbox_html).val().trim().replace(/,\s+/g,",").split(/[\n,\s+]/);
```

Comment:
On a second thought, it would be better to standardize the email list to be comma-separated, as stated in the ticket, and also add a tooltip to indicate that the e-mails must be comma-separated.